core/store/src/main/java/uk/gov/gchq/gaffer/store/Store.java

```diff
@@ -451,21 +451,21 @@ public void validateSchemas() {
             validateSchema(validationResult, getSchema().getVertexSerialiser());
 
             getSchema().getTypes().entrySet().forEach(entrySet ->
-                    validateSchema(validationResult, entrySet.getValue().getSerialiser()));
+                                                              validateSchema(validationResult, entrySet.getValue().getSerialiser()));
```

Comment:
I've been using the zip that was given to me from @p013570 unless mine has drifted away.
I'll check, against what you have.